playground/parseMson.js

```diff
@@ -1,8 +1,19 @@
-import get from 'lodash/get';
-import drafterjs from 'drafter.js';
+const fury = require('fury');
+const apibParser = require('fury-adapter-apib-parser');
+const minim = require('minim');
+const parseResultNamespace = require('minim-parse-result');
+
+const minimNamespace = minim.namespace();
+
+fury.use(parseResultNamespace);
+fury.use(apibParser);
+
 
 export default function parseMson(mson, cb) {
-  drafterjs.parse(mson.trim(), {}, (err, parseResult = {}) =>
-    cb(err, get(parseResult, 'content[0].content[0].content', []).map(el => el.content[0]))
-  );
+  fury.parse({ source: `FORMAT: 1A\n${mson.trim()}` }, (err, parseResult = {}) => {
+    let dataStructures;
+    dataStructures = minimNamespace.toRefract(parseResult.api.dataStructures.first);
```

Comment:
The `fury` module/object provides a pre-configured minim namespace that was used in parsing. You could use `fury.minim.serialise(parseResult.api.dataStructures.first)` instead of creating the minimNamespace and using `toRefract`

Reply:
Actually, since the purpose of the PR is to make attributes kit accept element object directly. What about just passing it down the data structure element directly to callback instead of serialising the element?

```js
cb(null, parseResult.api.dataStructures.first)
```